modules/data/com.spacestation14.Launcher.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Space Station 14
+Comment=Multiplayer disaster simulator
+Exec=/app/ss14/bin/SS14.Launcher
```

Comment:
the executable should in /app/bin (even via a symlink)  and the Exec line not have an absolute path.


also the desktop file should be submitted upstream as a desktop app for Linux without .desktop, it's a bug.